tests/__snapshots__/Rely_array_matchers_toBeEmpty.2af8e875.0.snapshot

```diff
@@ -0,0 +1,13 @@
+Rely array matchers › toBeEmpty › expect.array.not.toBeEmpty output
+Running 1 test suite
+\r\027[33m\027[1m\027[7m RUNS \027[27m\027[22m\027[39m \027[97mexpect.array.not.toBeEmpty output\027[39m\027[78D\027[K\027[31m\027[1m\027[7m FAIL \027[27m\027[22m\027[39m \027[97mexpect.array.not.toBeEmpty output\027[39m
```

Comment:
100% agree, I've been thinking about this for some time. I think the optimal solution is to build in support for Pastel for a mode that is more like your latter suggestion (such a think exists for JS chalk already I believe?)

Without changing Pastel, we probably want to add a configuration option to pass in a module with a signature pretty close to Pastel. Right now Rely is pretty tightly coupled to Pastel, and reducing this would be awesome for supporting reporters in non terminal environments.

My guess is we likely want to do both so that Pastel isn't referenced explicitly in matchers and formatting is pretty elsewhere and also to support the latter example you provided, which is something that I have found myself wanting in multiple different projects.

Reply:
I looked deeper into how this works: basically chalk has a notion of modes (for different levels of terminal color support) as well as an enabled flag, that can be used to basically turn all the chalkers into the identify function.

Jest has a serializer that turns ansi into the nice <Green></Green> tags. I think that it would be less hacky to build that in by default.

This has been bothering me for awhile so I'm going to spend a bit of time today playing with this separately to see if I can find an API I like